pkg/agent/flowexporter/connections/connections.go

```diff
@@ -37,8 +38,20 @@ var serviceProtocolMap = map[uint8]corev1.Protocol{
 	132: corev1.ProtocolSCTP,
 }
 
-type ConnectionStore struct {
+type ConnectionStore interface {
+	// Run enables the periodical polling of conntrack connections at a given flowPollInterval.
+	Run(stopCh <-chan struct{})
+	// GetConnByKey gets the connection in connection map given the connection key.
+	GetConnByKey(flowTuple flowexporter.ConnectionKey) (*flowexporter.Connection, bool)
+	// ForAllConnectionsDo execute the callback for each connection in connection map.
+	ForAllConnectionsDo(callback flowexporter.ConnectionMapCallBack) error
+	// DeleteConnectionByKey deletes the connection in connection map given the connection key.
+	DeleteConnectionByKey(connKey flowexporter.ConnectionKey) error
```

Comment:
when is this method called? I can't find it in the diff

Reply:
Please see L305 in exporter.go file.